common/lib/xmodule/xmodule/tests/test_library_content.py

```diff
@@ -23,6 +26,23 @@
 dummy_render = lambda block, _: Fragment(block.data)  # pylint: disable=invalid-name
 
 
+class TestHelperFunctions(TestCase):
+    """
+    Test class for library_content_module helper functions.
+    """
+    def test_sample(self):
+        """
+        Tests that _sample pulls relatively uniformly from a population.
+        """
+        population = list(range(10))
+        selections = Counter()
+        for _ in xrange(1000):
+            selections[_sample(population, 1)[0]] += 1
+
+        for element in population:
+            self.assertTrue(20 < selections[element] < 200)
```

Comment:
I get what you're going here, but this reads like a test of the random module, not of our code.

Reply:
Yeah, that thought crossed my mind while writing the test.  Maybe this doesn't need a test?  Or something simpler, like:
```python
selected = _sample(population, n)
for item in selected:
    assert selected in population
```

Reply:
Honestly, if you just put it the random object in the code and did a straight call to sample() like:

```python
rand = random.Random()
added_block_keys = set(rand.sample(pool, num_to_add))
```

I don't think it even needs a test. Because at that point you're really only testing that the stdlib function does what it says it does.

Reply:
Fair point.  Done.